### PR TITLE
Update extend.md

### PR DESCRIPTION
**Pull request by byog**
## Checklist:
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

The sentence “Include the following meta tag will unintrusively allow the user the option to download your iOS app” in extend.md lets me confused, I think the sentence doesn't quite make sense.I'd suggest:
> Including the following meta tag will unintrusively give the user the option to download your iOS app

and update the invalid link about `meta tag` in it to:
https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariWebContent/PromotingAppswithAppBanners/PromotingAppswithAppBanners.html

### CODE REVIEW

File: src/doc/extend.md [hunk 1]
```diff
@@ -25,9 +25,7 @@ not everything fits with everyone's needs.
 ### Smart App Banners in iOS 6+ Safari
 
 Stop bothering everyone with gross modals advertising your entry in the
-App Store. Include the following [meta tag](https://developer.apple.com/library/IOS/documentation/AppleApplications/Reference/SafariWebContent/PromotingAppswithAppBanners/PromotingAppswithAppBanners.html#//apple_ref/doc/uid/TP40002051-CH6-SW2)
-will unintrusively allow the user the option to download your iOS app,
-or open it with some data about the user's current state on the website.
+App Store. Including the following [meta tag](https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariWebContent/PromotingAppswithAppBanners/PromotingAppswithAppBanners.html) will unintrusively give the user the option to download your iOS app,or open it with some data about the user's current state on the website.
```
Comment (roblarsen) [inline src/doc/extend.md]: "unintrusively" > "unobtrusively"

Comment (byog) [reply src/doc/extend.md]: You are right.

Comment (byog) [reply src/doc/extend.md]: Do I need to close this PR and open a new PR for this?

Comment (roblarsen) [reply src/doc/extend.md]: Sorry, I was away for the weekend. @byog you can just push new code to your branch, You can keep the same PR.

Comment (byog) [reply src/doc/extend.md]: I have already modified it on my branch. What next?

Comment (roblarsen) [reply src/doc/extend.md]: @byog That's all it takes. Changes to a PR branch are automatically reflected in the PR.

**Comment by roblarsen** [comment]
Thanks @byog 

closes #2002

**Pull request merged**